incuna_auth/middleware/permission_feincms.py

```diff
@@ -53,7 +53,10 @@ def _get_resource_access_state(self, request):
 
         # Chase inherited values up the tree of inheritance.
         INHERIT = AccessState.STATE_INHERIT
-        while feincms_page.access_state == INHERIT and hasattr(feincms_page, 'parent'):
+        while (
+            feincms_page.access_state == INHERIT and
+            getattr(feincms_page, 'parent', None)
```

Comment:
https://github.com/feincms/feincms/blob/master/feincms/module/page/models.py#L177-L179

Looks like the Page does always have a parent.